app.controller/app.controller.js

```diff
@@ -0,0 +1,11 @@
+const { fetchTopics } = require("../app.model/app.model");
+
+exports.getTopicsRequest = (request, response, next) => {
+  return fetchTopics()
+    .then((topics) => {
+      return res.status(200).send(topics);
```

Comment:
The correct way in which to send dat back should be an object..not just the data. 

Please fix this so that your data is a property value on an object that can accessed via a key.